pcluster/pcluster_stack.py

```diff
@@ -31,14 +31,25 @@ def get_version_list(package_name):
     versions = data["releases"].keys()
     return list(versions)
 
+#def get_git_version_list(git_owner, package_name):
+#    import json
+#    import requests
+#    from distutils.version import StrictVersion
+#
+#    url = "https://api.github.com/repos/%s/%s/tags" % (git_owner, package_name,)
+#    data = requests.get(url).json()
+#    versions = [v['name'] for v in data]
+#    return list(versions)
+
+
 def get_git_version_list(git_owner, package_name):
     import json
-    import requests
-    from distutils.version import StrictVersion
+    import os
+    from github import Github
 
-    url = "https://api.github.com/repos/%s/%s/tags" % (git_owner, package_name,)
-    data = requests.get(url).json()
-    versions = [v['name'] for v in data]
+    g = Github(os.environ.get('GIT_AUTH_TOKEN'))
```

Comment:
Where does `GIT_AUTH_TOKEN` come from?

Reply:
Your github account. Settings -> Developer Settings -> Personal Access Tokens. (Dont check any boxes). Put it in your bashrc as
```
export GIT_AUTH_TOKEN=XXXXXXX
```

Reply:
Added doc to README for developers